src/components/image/image.stories.mdx

```diff
@@ -58,6 +58,17 @@ To hide the component completely - meaning hidden visually and from being access
   <Story name="with hidden prop" story={stories.WithHiddenProp} />
 </Canvas>
 
+### Decorative Image
+
+There may be a use case where a decorative image is required. A decorative images does not add information to the content of a page. 
```

Comment:
```suggestion
There may be a use case where a decorative image is required. A decorative images do not add information to the content of a page. 
```

Reply:
Probably should be:
`Decorative images do not add...`